apps/pipelines/nodes/nodes.py

```diff
@@ -394,16 +411,36 @@ def _create_model_from_data(value_data, model_name: str):
         return schema
 
 
-class ExtractStructuredData(ExtractStructuredDataNodeMixin, LLMResponse):
+class StructuredDataSchemaValidatorMixin:
+    @field_validator("data_schema")
+    def validate_data_schema(cls, value):
+        try:
+            parsed_value = json.loads(value)
+        except json.JSONDecodeError:
+            raise PydanticCustomError("invalid_schema", "Invalid schema")
+
+        if not isinstance(parsed_value, dict) or len(parsed_value) == 0:
+            raise PydanticCustomError("invalid_schema", "Invalid schema")
+
+        return value
+
+
+class ExtractStructuredData(ExtractStructuredDataNodeMixin, LLMResponse, StructuredDataSchemaValidatorMixin):
     __human_name__ = "Extract Structured Data"
     __node_description__ = "Extract structured data from the input"
-    data_schema: ExpandableText
+    data_schema: ExpandableText = Field(
+        default='{"name": "the name of the user"}',
+        description="A key-value pair where the key is the name of the field and the value the description",
```

Comment:
This doesn't have to be just key-value pair:
```suggestion
        description="A JSON data structure with descriptions for each object key.",
```

Reply:
Technically it does have to be a dict-like object. We don't support lists at the top-level currently. I can't remember why exactly, but thinking about it, what would that actually mean and what schema do we give the LLM?

Reply:
That's fine, maybe we can say a `JSON object structure`?

Reply:
Sure, I'll update. I see you've approve, so Ima merge once I updated. Thanks!

Reply:
https://github.com/dimagi/open-chat-studio/pull/857/commits/e828ec811ea4cded658edecabb4c1480417af5a9